lib/setupmanagers/qemuhck/qemuhck.rb

```diff
@@ -108,6 +110,40 @@ def initialize_clients_vm
       end
     end
 
+    def write_host_file
+      host_log = "#{@workspace_path}/#{HOST_INFO_LOG_FILE}"
+      File.write(host_log, collect_environment_info)
+      @project.result_uploader.upload_file(host_log, HOST_INFO_LOG_FILE)
+    end
+
+    def collect_environment_info
+      logs = String.new
+      compose_host_info(logs)
+      compose_vms_info(logs)
+      logs
+    end
+
+    def compose_host_info(logs)
+      logs << <<~HOST_INFO
+        QEMU version: #{`qemu-system-x86_64 --version`.lines.first.strip}
+        System information: #{`uname -a`.strip}
+      HOST_INFO
+    end
+
+    def compose_vms_info(logs)
+      logs << <<~STUDIO_INFO
+        Studio Properties:
+          #{@studio_vm.dump_config}
+
```

Comment:
For consistency do either:
- add a blank line after the host information.
- Remove the blank line after the studio properties.